core/services/scan.go

```diff
@@ -280,7 +282,8 @@ func generateScanID(workload domain.ScanCommand) string {
 		return workload.InstanceID
 	}
 	if workload.ImageTag != "" && workload.ImageHash != "" {
-		return workload.ImageTag + workload.ImageHash
+		sum := sha256.Sum256([]byte(workload.ImageTag + workload.ImageHash))
```

Comment:
@dwertent fixed

Reply:
Please use FNV. it's much cheaper and we have no need in cryptography grade hash function in this scenario